src/tenants/salesforce/TreeCounter/TreeCounter.tsx

```diff
@@ -3,11 +3,12 @@ import treeCounterStyles from './TreeCounter.module.scss';
 import { getFormattedNumber } from '../../../utils/getFormattedNumber';
 
 export default function TpoProfile(props: any) {
+  console.log(parseInt('5'))
   return (
     <div className={`${treeCounterStyles.treeCounter} ${props.planted > 0 ? treeCounterStyles.treeCounterReady : ''}`}>
       <div className={treeCounterStyles.treeCounterDataField}>
         <span aria-label={props.planted} role="heading">
-          {String(getFormattedNumber('en', Number(props.planted))).padStart(11).split("").map(function(char, index){
+          {String(getFormattedNumber('en', parseInt(props.planted))).padStart(11).split("").map(function(char, index){
```

Comment:
I think this does not work as the commas are inserted by our `getFormattedNumber` helper method. I tested it locally with `npm run dev` at http://localhost:3000/home - you have to replace your default values with this in your .env.local file before running to see the custom tenant output:
```
TENANT=salesforce
TENANTID=ten_3hEjJCBs
```

Reply:
Oh, sorry, I was wrong about the business requirements. They just wanted to remove the decimals.